scripting/erm/ERMParser.cpp

```diff
@@ -187,7 +187,7 @@ std::vector<LineInfo> ERMParser::parseFile(CERMPreprocessor & preproc)
 	}
 	catch (ParseErrorException & e)
 	{
-		logGlobal->error("Stopped parsing file.");

```

Comment:
It causes  warning because 'e' is not used

Reply:
logGlobal->error("Stopped parsing file. %s", e.what()); Also we can log file name if available.

Reply:
Ok